lib/Server.js

```diff
@@ -120,6 +119,7 @@ function Server(compiler, options) {
 
 		proxy: function() {
 			if (options.proxy) {
+				var proxy = new httpProxy.createProxyServer({secure: options.secureProxy});
```

Comment:
It's a dev-server, which is only used in development so we don't need to care about security that much. For me `secure: false` looks good.